os/se/ameba/rtl_security_api_wrapper_tz.c

```diff
@@ -33,8 +33,13 @@ typedef struct {
 	void (*info_printf)(const char *);
 } nsfunc_ops_s;
 
-/* Flash Status Bit */
+/* Flash Status Bit Protect Lower 512KB */
+#ifdef CONFIG_AMEBAD_TRUSTZONE
 #define FLASH_STATUS_BITS 0x2c
+#endif
+#ifdef CONFIG_AMEBASMART_TRUSTZONE
+#define FLASH_STATUS_BITS 0x28
+#endif
```

Comment:
I don't know this is proper. I don't have any requirement change on this. But anyway, when we want to change the region, we should modify several files. And it could be mismatched.
How about making this as a config?

And here are amebad and amebasmart only. How about amebalite? Because we support it, I ask you this.

Reply:
Different IC packaging uses different Flash, it has different Status bits values.
We will base it on previous AmebaLite packaging, thus it's the same as AmebaD.

We will create another PR to change this setting to a config, is that okay?

Reply:
> Different IC packaging uses different Flash, it has different Status bits values.
> We will base it on previous AmebaLite packaging, thus it's the same as AmebaD.

Because of that, I suggest making a config to set for different boards configuration.

> We will create another PR to change this setting to a config, is that okay?

Sure